### ISSUE TITLE
Disable positional args linter for native.register_toolchains

### ISSUE THREAD

**Issue by Monnoroch** [bazelbuild/buildtools#456]
`register_toolchains` rule accepts a python's argument list, which doesn't have argument names.

[Linter](https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#positional-args), [rule](https://docs.bazel.build/versions/master/skylark/lib/globals.html#register_toolchains).

**Comment by laurentlb** [comment]
I think the check should be enabled only for BUILD files.

**Comment by Monnoroch** [comment]
@laurentlb not necessarily. Rules like `http_archive` have named arguments and using them certainly improves readability.

**Comment by Monnoroch** [comment]
A perfect solution would be to read the macro's signature, determine if it has `*args` and only disable the lint for those.
